src/_data/schemas/lpa.json

```diff
@@ -0,0 +1,289 @@
+{
+    "$schema": "http://json-schema.org/schema",
+    "$ref": "#/definitions/LPA",
+    "definitions": {
+        "LPA": {
+            "type": "object",
+            "description": "The main schema for the Lasting Power of Attorney Document.",
+            "additionalProperties": false,
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "format": "integer",
```

Comment:
I think we probably want to err towards UUID or similar for unique ids on this kind of schema. Flat integers suffer from numeric guessing attacks (i.e. if you know LPA 1 exists you can guess that 2,3,4 etc exist) which can impact API attack surfaces when you expose the data.

Reply:
Further thought. Some kind of urn like `urn:opg:lpa:<uuid>` might be worth thinking about for interoperability and data sharing. https://en.wikipedia.org/wiki/Uniform_Resource_Name#